tasks/data_handling/tests/test_data_pre_processing.py

```diff
@@ -0,0 +1,283 @@
+import scanpy as sc
+import unittest
+import os
+import numpy as np
+from collections import Counter
+
+import global_settings as gs
+from tasks.data_handling import data_pre_processing
+from tasks.test_helper.builders.data_handling import data_pre_process as test_data_pre_process
+from tasks.test_helper.builders.utils import context_utils as cu
+
+
+class TestCreateCountMatrix(unittest.TestCase):
+    def test_need_file_path(self):
+        with self.assertRaises(ValueError) as context:
+            result = data_pre_processing.create_count_matrix()
```

Comment:
here you should also get the path where the simulated .h5 was saved

Reply:
I'm not sure what you mean, do you want me to define the file path before `test_need_file_path()`?